internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
```

Comment:
The function is directly passing a context (`ctx`) to the `GithubController` and the routes. Ensure that the context is being used appropriately within the handlers, as improperly using context can lead to cancellation or timeout issues. Consider whether it is essential to pass a context parameter to the controller or if each handler could create its context locally as needed.